### PR TITLE
ntkgp option in predict.gp_inference

### PR DESCRIPTION
**Pull request by bobby-he**
added `get='ntkgp'` option in nt.predict.gp_inference and also added test in tests.predict_test.testGpInference

### CODE REVIEW

**Comment by bobby-he** [comment]
@googlebot I signed it!
________________________________

**Review by romanngg — CHANGES_REQUESTED** [review]
Thanks a lot for the change, it looks great! Left some minor comments, the main one regarding whether it could support `get=('ntk', 'nngp', 'ntkgp')`, lmk what you think!

File: neural_tangents/predict.py [hunk 1]
```diff
@@ -559,7 +559,7 @@ def k_inv_y(g: str):
   @utils.get_namedtuple('Gaussians')
   def predict_fn(get: Get=None,
                  k_test_train=None,
-                 nngp_test_test: np.ndarray = None
+                 k_test_test: np.ndarray = None
```
[the remark this answers is not in the document]
Comment (bobby-he) [reply neural_tangents/predict.py]: Yeah good idea! I just added some changes that allow `k_test_test` to be an array/namedtuple/kernel and a few more that test when get is a tuple containing `ntkgp`. Lmk what you think!

[the remark this answers is not in the document]
Comment (romanngg) [reply neural_tangents/predict.py]: Great job, thank you for the contribution!

**Pull request merged**